lib/libevent/libevent.nit

```diff
@@ -470,20 +510,33 @@ class ConnectionFactory
 	do
 		var base = listener.base
 		var bev = new NativeBufferEvent.socket(base, fd, bev_opt_close_on_free)
-
-		# Human representation of remote client address
-		var addr_len = 46 # Longest possible IPv6 address + null byte
-		var addr_buf = new CString(addr_len)
-		addr_buf = addrin_to_address(addrin, addr_buf, addr_len)
-		var addr = if addr_buf.address_is_null then
-				"Unknown address"
-			else addr_buf.to_s
-
+		var addr=""
+
+		if not listener isa UnixConnectionListener then
+			# Human representation of remote client address
+			var addr_len = 46 # Longest possible IPv6 address + null byte
+			var addr_buf = new CString(addr_len)
+			addr_buf = addrin_to_address(addrin, addr_buf, addr_len)
+			addr = if addr_buf.address_is_null then
+					"Unknown address"
+				else addr_buf.to_s
+		end
 		var conn = spawn_connection(bev, addr)
 		bev.enable ev_read|ev_write
 		bev.setcb conn
 	end
 
+	# Listen on domain & file for new connection, which will callback `spawn_connection`
+	fun bind_to_unix(hostname: String, file: String): nullable ConnectionListener
+	do
+		var listener = new UnixConnectionListener.bind_to(event_base, hostname.to_cstring, file.to_cstring, self)
+		if listener.address_is_null then
+			sys.stderr.write "Unix libevent warning: Opening {hostname}::{file} failed\n"
```

Comment:
Remove the "Unix " part frm the prefix please to keep a uniform output. You may add a "UNIX socket" elsewhere after the prefix if you think it can be useful.